core/migrations/0018_create_region_location_role.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 3.2.17 on 2023-02-20 13:01
+import logging
+from functools import lru_cache
+
+from django.db import migrations
+from core.models import RoleRight, Role
+
+ROLE_RIGHT_ID = 121906
+IMIS_ADMINISTRATOR_ROLE_IS_SYSTEM = 64
+logger = logging.getLogger(__name__)
+
+
+@lru_cache(maxsize=1)
```

Comment:
It's good to think about efficiency however I don't think there's a need for cache in here 
Roles table is not big and even in case of efficiency issues creating index on is_system would be better globally.